pandas/tests/frame/conftest.py

```diff
@@ -127,6 +127,17 @@ def timezone_frame():
     return df
 
 
+@pytest.fixture
```

Comment:
for now let's move to the module; as I commented elsewhere. If you find that you now have 2+ uses of the same fixtures, THEN let's pull them out.

Reply:
Fair enough, for *new* fixtures, I'll add them in their respective modules